maps/xoq.dm

```diff
@@ -0,0 +1,41 @@
+#ifndef MAP_OVERRIDE
+//**************************************************************
+// Map Datum -- xoqstation
+//**************************************************************
+
+/datum/map/active
+	nameShort = "xoq"
+	nameLong = "noitatS xoq"
```

Comment:
One of these words is rotated, one is reversed
It's fucked up

Reply:
well I cant rotate station OKAY, I'M NOT A GOD

Reply:
uoıʇɐʇs

Reply:
doesn't work in the .dm file sadly, just gives ??

Reply:
It would in 513 but sadly we haven't updated yet